docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.general.scss

```diff
@@ -50,15 +50,37 @@ $hb-spacing-list: (
   )
 );
 
+$hc-animation-duration: 150ms;
```

Comment:
I'd just put these in the array. I also felt the inclination to make spacing variables like this originally, but I think it's better if it's all just in the array (it's cleaner/neater)